lib/model.js

```diff
@@ -3009,6 +2974,9 @@ Model.findOneAndRemove = function(conditions, options, callback) {
  * @param {Boolean|String} [options.strict] overwrites the schema's [strict mode option](https://mongoosejs.com/docs/guide.html#strict)
  * @param {ClientSession} [options.session=null] The session associated with this query. See [transactions docs](/docs/transactions.html).
  * @param {Object|String|String[]} [options.projection=null] optional fields to return, see [`Query.prototype.select()`](#query_Query-select)
+ * @param {Any} [options.sort] if multiple docs are found by the conditions, sets the sort order to choose which doc to update. TODO: Update type to proper type
+ * @param {Boolean} [options.rawResult] if true, returns the [raw result from the MongoDB driver](https://mongodb.github.io/node-mongodb-native/4.3/interfaces/ModifyResult.html)
+ * @param {Any} [options.select] sets the document fields to return. TODO: Update type to proper type
```

Comment:
`[Object|String]`

Reply:
should be done in 48e9ebc177991cf299e3597e1572292a01f913db